[BUG] Distribution build broken for 1.3.2 release.

**What is the bug?**
Our current distribution build pipeline is broken for 1.3.2 release because of an error on AD plugin.
It seems like the index is wrong for this line.
https://github.com/opensearch-project/anomaly-detection/blob/cc83942d8f951c17b1410c83934875780cdd517f/build.gradle#L20 
Here are the build error contexts: 
```
FAILURE: Build failed with an exception.

* Where:
Build file '/tmp/tmppylbjek7/anomaly-detection/build.gradle' line: 20

* What went wrong:
A problem occurred evaluating root project 'opensearch-anomaly-detection'.
> begin 0, end -1, length 5

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org/

BUILD FAILED in 2s
2022-04-30 01:52:57 ERROR    Error building anomaly-detection, retry with: ./build.sh manifests/1.3.2/opensearch-1.3.2.yml --component anomaly-detection
```

**How can one reproduce the bug?**
Steps to reproduce the behavior:

Try to run this in build repo: `./build.sh manifests/1.3.2/opensearch-1.3.2.yml --component anomaly-detection`

**What is the expected behavior?**
A clear and concise description of what you expected to happen.

**What is your host/environment?**
 - OS: [e.g. iOS]
 - Version [e.g. 22]
 - Plugins

**Do you have any screenshots?**
If applicable, add screenshots to help explain your problem.

**Do you have any additional context?**
Add any other context about the problem.

--- a/build.gradle
+++ b/build.gradle
@@ -16,8 +16,13 @@ import org.opensearch.gradle.testclusters.StandaloneRestIntegTestTask
 buildscript {
     ext {
         opensearch_group = "org.opensearch"
+        isSnapshot = "true" == System.getProperty("build.snapshot", "true")
         opensearch_version = System.getProperty("opensearch.version", "1.3.2-SNAPSHOT")
-        plugin_version = opensearch_version.substring(0, opensearch_version.indexOf("-"))
+        if (isSnapshot) {
+            plugin_version = opensearch_version.substring(0, opensearch_version.indexOf("-"))
+        } else {
+            plugin_version = opensearch_version
+        }
         // 1.2.0 -> 1.2.0.0, and 1.2.0-SNAPSHOT -> 1.2.0.0-SNAPSHOT
         opensearch_build = opensearch_version.replaceAll(/(\.\d)([^\d]*)$/, '$1.0$2')
         common_utils_version = System.getProperty("common_utils.version", opensearch_build)

